src/dashboard/dashboard.html

```diff
@@ -2,10 +2,17 @@
 
 <section class="container now-showing-section">
```

Comment:
To test the dashboard (since I couldn't change movies) I modified this line to be `<section class="container now-showing-section" ng-init="dashboardVm.currentMovie.trailerUrl = ''https://www.youtube.com/watch?v=v2DhIc8s4lY">` which I assumed would be the same as if the video actually had a movie trailer saved.  Now that I'm an admin I'll pull down your changes and re-test it.  If you call out the minor changes you'd like made I'll touch those up too.